docs/admin/release_notes/version_2.2.md

```diff
@@ -0,0 +1,27 @@
+
+# v2.2 Release Notes
+
+This document describes all new features and changes in the release. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
We generally highlight the new features in a slightly more human format here. Perhaps just 

- Added Python 3.12 support.
- Added REST API endpoint for Jinja as first part of Journey towards a jinja live editor.